install_files/ansible-base/roles/tails-config/files/securedrop_init.py

```diff
@@ -148,3 +149,70 @@
 if b'Update needed' in output or os.path.exists(flag_location):
     # Start the SecureDrop updater GUI.
     subprocess.Popen(['python3', path_gui_updater], env=env)
```

Comment:
@zenmonkeykstop observed this morning that the invocation of the SecureDrop updater here, *before* the new logic that follows checks for a Tails upgrade, means that we will be able to update this logic in a future SecureDrop release in the event that it breaks under a future Tails version.  (That's a mouthful, but I hope it's a clear statement of the *solution* to a risk we've worried about!)

Reply:
Yes, and it's also my hope that we can remove this entire section after a reasonable timeframe (I've suggested Jan 2022 in the comments)